### PR TITLE
Update project info and remove redundant info

### LINKED ISSUE #113: Update project description
> This project is an open source tool to convert TIMES model Excel input files to DD format ready for processing by GAMS. The intention is to make it easier for people to reproduce research results on TIMES models.

These statements ignore the **existence** of Veda - it is **subsumed** in TIMES. I just want to ensure that developers and users of this project are aware that Veda and TIMES are separate entities.

"TIMES model input files" are the DD files. The Excel files you are refering to are **Veda input files**. These Excel files contain an original syntax that introduced a new way of **creating** data - via rule-based processing of flexible formats. For example, the ability to identify nodes of a RES based on set membership, name, description, and topology and add/edit parameters - is Veda. This has absolutely nothing to do with TIMES. The project is replicating this ability using different software tools, without mentioning "Veda" anywhere, which implicitly makes all the data concepts of Veda a part of TIMES.

_Originally posted by @akanudia in https://github.com/etsap-TIMES/times-excel-reader/issues/109#issuecomment-1702094757_

### PR DESCRIPTION
**Pull request by olejandro**
Update project info. Close #113

### CODE REVIEW

**Comment by olejandro** [comment]
@siddharth-krishna what do you think about changes in this PR so far?

**Comment by olejandro** [comment]
Also any opinion on https://bttger.github.io/contributing-gen-web/? I am thinking to use their template to generate contributing guide and code of conduct.

**Comment by siddharth-krishna** [comment]
Thanks! Both look good to me. :)

**Comment by olejandro** [comment]
@siddharth-krishna any chance you could check whether any of the info in README should be moved to CONTRIBUTING?

File: LICENSE [hunk 1]
```diff
@@ -1,21 +1,22 @@
-    MIT License
+MIT License
 
-    Copyright (c) Microsoft Corporation.
+Copyright (c) 2022-2023 Microsoft Corporation.
+Copyright (c) 2023-2024 IEA Energy Technology Systems Analysis Programme.
```
Comment (SamRWest) [inline LICENSE:4]: Does the license really need copyright statements?

**Review by SamRWest — APPROVED** [review]
Nice one.  I was wondering if the MS stuff was still relevant.
Just one minor query, otherwise looks good to me.

File: LICENSE (hunk 1 above)
Comment (olejandro) [reply LICENSE:4]: Normally a license needs a copyright statement. Here is an article about it on Wikipedia: https://en.wikipedia.org/wiki/MIT_License

Comment (SamRWest) [reply LICENSE:4]: Oh right, I didn't know that. Cool :)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+> If you want to ask a question, we assume that you have read the available [Documentation](https://xl2times.readthedocs.io/).
+
+Before you ask a question, it is best to search for existing [Issues](https://github.com/etsap-TIMES/xl2times/issues) that might help you. In case you have found a suitable issue and still need clarification, you can write your question in this issue. It is also advisable to search the internet for answers first.
+
+If you then still feel the need to ask a question and need clarification, we recommend the following:
+
+- Open an [Issue](https://github.com/etsap-TIMES/xl2times/issues/new).
+- Provide as much context as you can about what you're running into.
+- Provide project and platform versions (nodejs, npm, etc), depending on what seems relevant.
```
Comment (siddharth-krishna) [inline CONTRIBUTING.md:47]:
```suggestion
- Provide project and platform versions (at least: OS, python), depending on what seems relevant.
```

**Review by siddharth-krishna — APPROVED** [review]
My feeling is for now, while the tool user base and README are still quite small, it is better to keep the "Development" section in the README instead of moving it to CONTRIBUTING.md so that it is more obvious to new user/developers. We can move it to a separate file if the README gets too big?

File: README.md [hunk 3]
```diff
@@ -137,25 +138,6 @@ python -m build
...
-contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
-
-## Trademarks
-
-This project may contain trademarks or logos for projects, products, or services. Authorized use of Microsoft
-trademarks or logos is subject to and must follow
-[Microsoft's Trademark & Brand Guidelines](https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/usage/general).
-Use of Microsoft trademarks or logos in modified versions of this project must not cause confusion or imply Microsoft sponsorship.
-Any use of third-party trademarks or logos are subject to those third-party's policies.
+This project welcomes contributions and suggestions.
```
Comment (siddharth-krishna) [inline README.md:143]:
```suggestion
This project welcomes contributions and suggestions. See [CODE_OF_CONDUCT.md](CODE_OF_CONDUCT.md) and [CONTRIBUTING.md](CONTRIBUTING.md) for more details.
```

Comment (siddharth-krishna) [reply README.md:143]: Perhaps we move the `## Contributing` block above the `## Development` block, and have it say details for how to set up the project for development are below?

**Review by siddharth-krishna — COMMENTED** [review]
Also, a gentle reminder not to force push PR branches once they have been reviewed, as it makes it more difficult to see what has changed when doing a second review.

**Pull request merged**